plugins/experimental/slice/Range.h

```diff
@@ -29,7 +29,7 @@
 
 struct Range {
 public:
-  static int64_t constexpr maxval = (std::numeric_limits<int64_t>::max() >> 2);
+  static int64_t constexpr maxval = (std::numeric_limits<int64_t>::max() / 2);
```

Comment:
It's not clear why this is being changed from max / 4 to max / 2, or really why it's not just max.  But I'll take it on faith.

Reply:
it just has to be sufficiently large but not max64.